src/ffi/client_manager.js

```diff
@@ -8,9 +8,26 @@ const _clientHandle = Symbol('clientHandle');
 
 class ClientManager extends FfiApi {
   static manifest = {
-    setNetworkListener: 'sync'
+    setNetworkListener: 'sync',
+    login: 'promise',
+    createAccount: 'promise'
   };
 
+  static _isUserCredentialsValid(locator, secret) {
+    if (typeof locator !== 'string' || typeof secret !== 'string') {
+      return { isValid: false, msg: 'Locator or Secret must be of string' };
+    }
+
+    const userLocator = locator.trim();
+    const userSecret = secret.trim();
+
+    if (!(userLocator && userSecret)) {
+      return { isValid: false, msg: 'Locator or Secret should not be empty' };
```

Comment:
Can we integrate [i18n](https://www.npmjs.com/package/i18n) as a part of this PR?